sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/query.sql

```diff
@@ -215,7 +188,7 @@ final_staging AS (
     engmgt.first_interaction,
     CAST(engmgt.last_interaction AS float64) AS last_interaction,
     all_events.is_entrance AS entrances,
-    COALESCE(exits.is_exit, 0) AS exits,
+    COALESCE(CASE WHEN exits.max_row_number = all_events.row_nbr THEN 1 ELSE 0 END, 0) AS exits,
```

Comment:
Why does this have `exits` and `is_exit`?

Reply:
Got it. In that case I'd move it to the final `SELECT` statement, so we don't have to define the same logic twice:
```
SELECT
  final.is_exit,
  final.is_exit AS exits,
```